etc/kayobe/kolla.yml

```diff
@@ -329,7 +329,7 @@ kolla_build_blocks:
     {% endfor %}
   footer: |
     {% if stackhpc_kolla_clean_up_repo_mirrors | bool %}
-    {% if kolla_base_distro == 'centos' %}
+    {% if kolla_base_distro in ['centos', 'rocky'] %}
```

Comment:
From Michal
> `if base_package_type == 'rpm'`  would be probably better

But it might be better to make that a separate change

Reply:
But that would need to go as raw to Kolla-build